SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -21,20 +21,11 @@ ServerTreeView {
 		if(parent.isNil) {
 			window = Window(this.asString, bounds, scroll:true);
 		} {
-			if(parent.view.asString == "a TopView") {
-				var name = parent.name;
-				(
-					"The Window should be created with scroll argumrnt value true. e.g.:" ++
-					"Window.(" ++ "title".quote ++ ", scroll: true)"
-				).postln;
-				window = Window(parent.name, bounds, scroll:true);
-				parent.close;
-			} {
-				window = parent;
-			}
+			window = parent
 		};
 		viewParent = window;
-		window.view.hasHorizontalScroller_(false).background_(Color.grey(0.9));
+		if(window.view.respondsTo(\hasHorizontalScroller_)) { window.view.hasHorizontalScroller_(false) };
```

Comment:
thinking about it – perhaps we don't need the test. Doesn't the view of a window always respond to `hasHorizontalScroller_` ?

Reply:
I think we need the test.
Please watch the following video:
https://www.dropbox.com/scl/fi/mqns39chx08rbobhbyiew/hasHorizontalScroller_.mov?rlkey=7ymra5gy3nk5kae6tdo5z3kvz&dl=0

Reply:
ah good, I see. You could have told me directly :) – only subclasses of `AbstractScroll` have it …

Reply:
Thinking about it: if someone hands in a parent that has horizontalScorll on, why not? I'd try not to change something from the inside.

Reply:
OK, I removed the line:
```
if(window.view.respondsTo(\hasHorizontalScroller_)) { window.view.hasHorizontalScroller_(false) };
```
https://github.com/supercollider/supercollider/pull/6294/commits/dd7d58932f56d008acd43ad8a3d0615593f2b486

>Thinking about it: if someone hands in a parent that has horizontalScorll on, why not? 

To avoid confusion:
I did not add the following line:
```
window.view.hasHorizontalScroller_(false).background_(Color.grey(0.9));
```
It was already there when I saw this file to edit.